admin/server-timing.php

```diff
@@ -71,12 +78,16 @@ static function() {
 					<?php
 					echo wp_kses(
 						sprintf(
-							/* translators: 1: Server-Timing, 2: template_include */
-							__( 'Since the %1$s header is sent before the template is loaded, only hooks before the %2$s filter can be measured.', 'performance-lab' ),
+							/* translators: 1: Server-Timing, 2: template_include, 3: anchor link */
+							__( 'Since the %1$s header is sent before the template is loaded, only hooks before the %2$s filter can be measured. Enable <a href="%3$s">Output Buffering</a> to measure hooks during template rendering.', 'performance-lab' ),
```

Comment:
Similar to my similar comments, can you split those two sentences and use a `<br>` between them to avoid long lines?

Reply:
How about instead of adding line breaks, we instead add a max-width to the container? Adding line breaks can make it look bad on small viewports too:

![image](https://github.com/WordPress/performance/assets/134745/d3e200a9-7ff6-4575-80f1-aece316e0996)

Reply:
See 37d775aa86582a470b2fb1ca9abb44e3213b5353:

![image](https://github.com/WordPress/performance/assets/134745/126f3b53-98d2-40b4-b0d2-9c7909fd3f0e)

![image](https://github.com/WordPress/performance/assets/134745/ac22f4c5-9ff2-40af-9df4-f940d5434b12)

Reply:
Note a 800px max-width is also the approach that the Privacy Policy settings screen takes.

Reply:
Thanks, I like that idea! 👍